apps/website/src/content/docs/table.mdx

```diff
@@ -6,16 +6,466 @@ thumbnail: Table
 
 <p>{frontmatter.description}</p>
 
-<Placeholder componentPageName='table--basic' />
+Tables are an important element in most web-based applications. We’ve presented a set of flexible standards to allow for a variety of table configurations. This flexibility enables you to configure your `Table` with the necessary functionality without over-complicating the design.
 
 <LiveExample src='Table.main.jsx'>
   <AllExamples.TableMainExample client:load />
 </LiveExample>
 
-Tables are an important element in most web-based applications. We’ve presented a set of flexible standards to allow for a variety of table styles. As always, the specifics of how you style your table should be determined by the desired user experience and the goals of the application.
-
 Bentley makes extensive use of tables and data grids throughout its web applications. Use the following flexible grid format below in most circumstances as it provides functionality “built in” to the design, and because users learning how to consistently sort, filter, and search tables is an important skill to leverage.
 
+## Basic Usage
+
+Related sections: [Commonly Used Props](#commonly-used-props), [All Props](#props).
+
+**Note**: To avoid bugs, always try **memoizing** props.
```

Comment:
"Related sections" could be moved at the end, after the basic demo?

And this memoization "note" can be moved after the bullet list but before the demo.